nservicebus/eventstoretransport/outbox_diagram_eventstore_[,2).partial.md

```diff
@@ -0,0 +1,26 @@
+```
```

Comment:
@weralabaj it should be but this one is not included in the main doco (yet) so it does not cause build errors ;-)